sdk/go/common/resource/config/testdata/repr/0.yaml

```diff
@@ -0,0 +1,2 @@
+my:anotherTestKey: anotherTestValue
```

Comment:
Can these named more descriptively than just an index, rather not have 75+ files with no indication of what part of config they're testing.

Reply:
I would love it if these had more descriptive names, but I generated these from the set of unique `config.Map` values used by other tests. So I don't really have descriptive names to give them.